tests/core/filter/manual/classcheckinlexicalorder.html

```diff
@@ -0,0 +1,19 @@
+<textarea id="classic"><p>Hello world</p></textarea>
+<div id="inline" contenteditable="true"><p>Hello world</p></div>
+
+<script>
+	CKEDITOR.replace( 'classic', {
+		stylesSet: [
```

Comment:
When each pair of key and value is in new line it is much more easier to read. Also I'd consider extracting styleSet to variable, as it is used more than once.